change/@ni-nimble-components-bcea55bf-ed59-4b11-b80b-6311749da460.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@atmgrifter00 @rajsite I'm finding it difficult to review this PR now since it's adding several hundred lines of FAST code with custom implementations sprinkled in throughout. Should we break it into separate PRs? Annotate what's custom and what's copied? Any other suggestions?

Reply:
That change was a result of the discussion in this thread: https://github.com/ni/nimble/pull/1762#discussion_r1475489622

Summarized as we should own the code instead of overriding tons of Foundation methods. But we should sync up.

Reply:
Talked at standup and meyers is adding some PR comments to guide what changes are adopted from the fast impl

Reply:
The comments were really helpful, thank you.